src/helpers/MiscFunctions.cpp

```diff
@@ -609,14 +611,24 @@ void logSystemInfo() {
 
     Debug::log(NONE, "\n");
 
-#if defined(__DragonFly__) || defined(__FreeBSD__)
-    const std::string GPUINFO = execAndGet("pciconf -lv | fgrep -A4 vga");
-#elif defined(__arm__) || defined(__aarch64__)
-    const std::string GPUINFO = execAndGet("cat /proc/device-tree/soc*/gpu*/compatible");
-#else
-    const std::string GPUINFO = execAndGet("lspci -vnn | grep VGA");
-#endif
+    int fd = open("/dev/dri/card0", O_RDONLY | O_CLOEXEC);
```

Comment:
same stuff here, also why are you duping code? make this into a function? renderer does the same, btw, idk why this is needed here